src/main/resources/templates/splash.html

```diff
@@ -29,5 +29,24 @@ <h2>ADD YOUR OWN ALBUM</h2>
         </fieldset>
     </form>
 
+    <h2>ADD SONGS TO ALBUM</h2>
+    <form action="/song/add/" method="POST">
+        <fieldset>
+            <legend>Add Track from Album</legend>
+            <label>Song Title</label>
+            <input type="text" name="title">
+            <label>Track Number</label>
+            <input type="number" name="trackNumber">
+            <label>Song Length (in seconds)</label>
+            <input type="number" name="songLength">
+            <label>Album Title</label>
+            <input type="text" name="albumTitle">
+<!--            <label>Album Cover Image URL</label>-->
+<!--            <input type="text" name="imgUrl">-->
+            <input type="hidden" name="album" th:value="${album}">
```

Comment:
This needs to have some way to pass the appropriate album with the form when it's submitted. How does the form know what album it's about? If you're using the album title, how does it know which album is associated with that title?